server/service/individualServices/AuthorizationApplication.js

```diff
@@ -254,7 +252,7 @@ exports.IsApplicationExists = async function (applicationaName, ReleaseNumber, a
     }
 }
 
-async function ApplicationandReleaseNumber(applicationaName, ReleaseNumber, authorization) {
+async function ApplicationandReleaseNumber(applicationaName, authorization) {
```

Comment:
Modify the function name as per the purpose of the functionality. From the underlying logic , "ApplicationandReleaseNumber" should not be the method name.